### PR TITLE
Contact style

### LINKED ISSUE #18: Style Contact Us page

### PR DESCRIPTION
**Pull request by bjariel**
image
Closes #18 Added styling for Contact page, iframe used for form. Background image is missing the scrolling behavior. Was not able to figure out how to implement using Tailwind, but is barely noticeable. I also added code so that the background image is not shown on smaller screens, as originally it was pulled to the top of the page, blocking the important content.

### CODE REVIEW

File: src/_includes/layouts/contact.liquid [hunk 1]
```diff
@@ -1,31 +1,29 @@
...
 layout: layouts/base.liquid
 ---
-<div class="bg-parallax-photo">
-  <div
-    :speed-factor="0.5"
-    sectionClass="parallax-outside"
-    containerClass="parallax-inside">
-    <img src="/assets/img/bg-triangle-blue2.png" class="float-right float-md-none" />
+<div class="relative hidden md:block">
+  <div class="absolute top-0 left-0 w-2/5 min-h-[21.875rem] max-w-full overflow-visible z-[-1] ">
```
Comment (tridnguyen) [inline src/_includes/layouts/contact.liquid:5]: I don't think you need this div on line 5.  Both divs on line 5 and 6 are position absolute.  When an element is position absolute, it's taken out of the flow of the DOM and are positioned to their closest relative parent (line 4).  So, the div on line 5 isn't really doing anything, unless you're setting a background image on it or doing something else with it in a stylesheet.  if that's the case, I wouldn't nest absolute positioned elements unless they have different relative parents.

**Comment by gracexu94** [comment]
@tridnguyen FYI I'm taking over this PR for Bella since she's on vacation! I ended up using flex for the entire structure of the page since I felt like it worked better, and I think it should also address your review comments. Would you mind taking another look if you have a minute? Thanks!

File: src/_includes/layouts/contact.liquid [hunk 2]
```diff
@@ -1,31 +1,29 @@
...
+    <div class="absolute top-0 left-0 right-0 h-auto w-full overflow-visible">
+      <img src="/assets/img/bg-triangle-blue2.png" class="float-left" />
```
[the remark this answers is not in the document]
Comment (randimays) [reply src/_includes/layouts/contact.liquid]: Re: accessibility for images

If the image is purely decorative, it should have a blank alt (`alt=""`) so the screen reader skips over it.

File: src/_includes/layouts/contact.liquid [hunk 3]
```diff
@@ -1,31 +1,19 @@
...
-<div class="bg-parallax-photo">
-  <div
-    :speed-factor="0.5"
-    sectionClass="parallax-outside"
-    containerClass="parallax-inside">
-    <img src="/assets/img/bg-triangle-blue2.png" class="float-right float-md-none" />
+<div class="flex flex-col sm:flex-row sm:justify-center md:justify-around">
+  <div class="py-5 px-4">
+    <h1 class="absolute text-blue px-4 md:mt-12 md:mb-12">{{ contactHeader }}</h1>
+     <img src="/assets/img/bg-triangle-blue2.png" alt="Blue triangle"/>
```
Comment (tridnguyen) [inline src/_includes/layouts/contact.liquid]: I just viewed the image for this, probably want to make the alt text more descriptive.  Something like, "Image of girl shaking hands."

File: src/_includes/layouts/contact.liquid (hunk 2 above)
[the remark this answers is not in the document]
Comment (gracexu94) [reply src/_includes/layouts/contact.liquid]: Thanks Randi, good callout! Updated in [this commit](https://github.com/kindsys/dsc-11ty-cms/pull/30/commits/526634dbd88fc9b1d04cb81fe98c035d1ee6c025)

File: src/_includes/layouts/contact.liquid [hunk 4]
```diff
@@ -1,31 +1,19 @@
...
-<div class="container py-5">
-  <div class="row justify-content-md-between">
-    <div class="col-md-6 py-md-5">
-      <h1 class="text-blue my-lg-5">{{ contactHeader }}</h1>
-    </div>
-
-    <div class="col-md-5 py-md-5">
-      <h5 class="text-gray">{{ content }}</h5>
-
+  <div class="flex flex-col px-4 items-center md:py-12 ">
```
Comment (tridnguyen) [inline src/_includes/layouts/contact.liquid:9]:
```suggestion
  <div class="px-4 items-center md:py-12 ">
```

I don't think the `.flex and .flex-col` are doing anything here.

**Review by tridnguyen — COMMENTED** [review]
@gracexu94 just one minor suggestgion.

**Review by tridnguyen — APPROVED** [review]

**Pull request merged**